src/bower_components/emby-webcomponents/htmlvideoplayer/plugin.js

```diff
@@ -1443,9 +1443,9 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
         var list = [];
 
         var video = document.createElement('video');
-        //if (video.webkitSupportsPresentationMode && video.webkitSupportsPresentationMode('picture-in-picture') && typeof video.webkitSetPresentationMode === "function") {
-        //    list.push('PictureInPicture');
-        //}
+        if (video.webkitSupportsPresentationMode && typeof video.webkitSetPresentationMode === "function") {
```

Comment:
Can you merge two `if`s in one?

Reply:
Yeah you definitely can, this will be fine.

Reply:
This is the way Apple describes this exact line for this function - and it has worked in my testing.

Reply:
Okay, I realized earlier in the Chat that I was misunderstanding the question here. I was only looking at the single highlighted line, and not looking further.

I have done some testing, and implemented some changes in https://github.com/jellyfin/jellyfin-web/pull/118/commits/c99f9c6d656e2259f90b2995074d11e78b5104d6. Let me know if that works.